src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,33 @@
+package com.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @NonNull
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @OneToMany
```
не використовуємо FetchType.EAGER

Reply:
ok, done with out EAGER